src/corehost/cli/json/casablanca/include/cpprest/asyncrt_utils.h

```diff
@@ -41,7 +41,7 @@
 #ifndef _WIN32
 //#include <boost/algorithm/string.hpp>
 #ifdef __GLIBC__
-#include <xlocale.h>
+#include <locale.h>
```

Comment:
From looking at `master`, this whole section can be removed.

See [https://github.com/dotnet/core-setup/pull/3129](https://github.com/dotnet/core-setup/pull/3129)

Reply:
Oh cool, I didn't see that commit. I ported it.